Panic in `MakeTerraformResult`

This code:
```typescript
import * as gcp from "@pulumi/gcp";

export function GetStaticIPAddress(
  name: string,
  project: string,
  region: string
) {
  return gcp.compute.getAddress({
    name: name,
    project: project,
    region: region
  });
}

GetStaticIPAddress("test", "abc", "us-central1").then(
  result => console.log(result.address)
);
```

Causes a panic in the GCP provider:

```
panic: runtime error: invalid memory address or nil pointer dereference
    [signal SIGSEGV: segmentation violation code=0x1 addr=0x30 pc=0x239f1ee]
    goroutine 42 [running]:
    github.com/pulumi/pulumi-gcp/vendor/github.com/pulumi/pulumi-terraform/pkg/tfbridge.MakeTerraformResult(0x0, 0xc0004ebd40, 0x0, 0x0)
        /home/travis/gopath/src/github.com/pulumi/pulumi-gcp/vendor/github.com/pulumi/pulumi-terraform/pkg/tfbridge/schema.go:415 +0x20e
    github.com/pulumi/pulumi-gcp/vendor/github.com/pulumi/pulumi-terraform/pkg/tfbridge.(*Provider).Invoke(0xc00039cff0, 0x2c4bf40, 0xc0007d9e00, 0xc0007de9b0, 0xc00039cff0, 0x1, 0x1)
        /home/travis/gopath/src/github.com/pulumi/pulumi-gcp/vendor/github.com/pulumi/pulumi-terraform/pkg/tfbridge/provider.go:803 +0x83b
    github.com/pulumi/pulumi-gcp/vendor/github.com/pulumi/pulumi/sdk/proto/go._ResourceProvider_Invoke_Handler.func1(0x2c4bf40, 0xc0007d9e00, 0x28e7920, 0xc0007de9b0, 0x2c4bf40, 0xc0007d9e00, 0x2c543a0, 0x3cb89a8)
        /home/travis/gopath/src/github.com/pulumi/pulumi-gcp/vendor/github.com/pulumi/pulumi/sdk/proto/go/provider.pb.go:1175 +0x86
    github.com/pulumi/pulumi-gcp/vendor/github.com/grpc-ecosystem/grpc-opentracing/go/otgrpc.OpenTracingServerInterceptor.func1(0x2c4bf40, 0xc0007d9e00, 0x28e7920, 0xc0007de9b0, 0xc0007db320, 0xc0007db340, 0x0, 0x0, 0x2c39180, 0xc0003e5e90)
        /home/travis/gopath/src/github.com/pulumi/pulumi-gcp/vendor/github.com/grpc-ecosystem/grpc-opentracing/go/otgrpc/server.go:57 +0x2b0
    github.com/pulumi/pulumi-gcp/vendor/github.com/pulumi/pulumi/sdk/proto/go._ResourceProvider_Invoke_Handler(0x2996100, 0xc00039cff0, 0x2c4bf40, 0xc0007d9b90, 0xc0007b2540, 0xc00042d260, 0x0, 0x0, 0x0, 0x0)
        /home/travis/gopath/src/github.com/pulumi/pulumi-gcp/vendor/github.com/pulumi/pulumi/sdk/proto/go/provider.pb.go:1177 +0x158
```

--- a/pkg/tfbridge/provider.go
+++ b/pkg/tfbridge/provider.go
@@ -801,7 +801,7 @@ func (p *Provider) Invoke(ctx context.Context, req *pulumirpc.InvokeRequest) (*p
 
 		// Add the special "id" attribute if it wasn't listed in the schema
 		props := MakeTerraformResult(invoke, ds.TF.Schema, ds.Schema.Fields)
-		if _, has := props["id"]; !has {
+		if _, has := props["id"]; !has && invoke != nil {
 			props["id"] = resource.NewStringProperty(invoke.ID)
 		}
 

--- a/pkg/tfbridge/schema.go
+++ b/pkg/tfbridge/schema.go
@@ -412,7 +412,7 @@ func MakeTerraformResult(state *terraform.InstanceState,
 	outMap := MakeTerraformOutputs(outs, tfs, ps, nil, false)
 
 	// If there is any Terraform metadata associated with this state, record it.
-	if len(state.Meta) != 0 {
+	if state != nil && len(state.Meta) != 0 {
 		metaJSON, err := json.Marshal(state.Meta)
 		contract.Assert(err == nil)
 		outMap[metaKey] = resource.NewStringProperty(string(metaJSON))

